<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>你帮我助物品交换</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="7">
     <widget class="QPushButton" name="btn_del">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btn_add">
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="7">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="rowCount">
           <number>1</number>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <column>
           <property name="text">
            <string>序号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>物品名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>物品说明</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>物品所在地址</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>联系人手机</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>邮箱</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="6" colspan="2">
     <widget class="QPushButton" name="exit">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Adobe Heiti Std</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>你帮我助物品交换软件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>admin</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn_search">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="le_search"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
